api/README.md

```diff
@@ -1,52 +1,73 @@
 # Kitty Items API
+The Kitty Items API is a RESTful API built with [express](https://expressjs.com/) that sends transactions to Flow using the [Flow JS SDK](https://github.com/onflow/fcl-js/tree/master/packages/sdk). It contains endpoints for both the [kitty-items](src/services/kitty-items.ts) & [storefront](src/services/storefront.ts) services to read & write data from both the Flow blockchain as well as a SQL DB.
 
-The Kitty Items API is a RESTful API built with [express](https://expressjs.com/) that sends transactions to Flow using the [Flow JS SDK](https://github.com/onflow/fcl-js/tree/master/packages/sdk).
+## Services
+Kitty Items uses main 4 services to function. These services are used to maintain data integrity between the blockchain & our database as well as allow avenues for users to interact with our data through exposed api endpoints. 
 
-## Setup
+### [Flow Service](src/services/flow.ts)
+This service contains functions that interact with the Flow blockchain using the [onflow/fcl](https://docs.onflow.org/fcl/) package. While it has no exposed endpoints, the methods here used to read, write & authorize data on the chain are used extensively in its sister services & workers.
 
-Run the commands below to initialize the minter account.
+Notable functions:
+- `sendTx()`: Takes a Cadence transaction as an arguement and sends it to write data into the Flow blockchain.
+- `executeScript()`: Takes a Cadence script as an arguement and executes it to read data from the Flow blockchain. 
+- `getLatestBlockHeight()`: Gets the most recent block from the Flow blockchain.
+- `authorizeMinter()`: Returns an asynchronous method to authorize an account to mint Kitty Items.
 
-- **POST /v1/kitty-items/setup** - Create a resource that holds Kitty Items in the `MINTER_FLOW_ADDRESS` account.
+### [Block-cursor Service](src/services/block-cursor.ts)
+This service contains functions that read & write data to the SQL database to maintain the Kitty Items Marketplace
 
-```sh
-curl --request POST \
-  --url http://localhost:3000/v1/kitty-items/setup \
-  --header 'Content-Type: application/json'
-```
+Notable functions:
+- `findOrCreateLatestBlockCursor()`: Queries the `block_cursor` table for the the last block check by the listing handler worker. If no row is found then insert a record with our current latest block height.
+- `updateBlockCursorById()`: Updates the row with the latest block height that has been checked for events.
 
-- **POST /v1/market/setup** - Create a resource that allows the `MINTER_FLOW_ADDRESS` to hold listings for Kitty Items.
+### [Kitty-items Service](src/services/kitty-items.ts)
```

Comment:
```suggestion
### [Kitty Items Service](src/services/kitty-items.ts)
```

Reply:
Also, I noticed that you took out the sample curl commands. I think those were helpful to show developers how to call the API.

Reply:
i took out the curl commands since we had flow scripts that sort of abstract away the endpoints but i can see the value in letting them hit the endpoints themselves. ill add the curl commands back in a `Getting Started` section